### PR TITLE
[T4A3][W14-A3]Loh Zi Bin, Robin

### PR DESCRIPTION
**Pull request by robinloh**

### CODE REVIEW

**Comment by robinloh** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -37,9 +36,6 @@ public static boolean isValidAddress(String test) {
     }
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Address.java:24]: How about considering pushing the constructor into the parent class too?

**Review by K1ang — COMMENTED** [review]
@robinloh Some comments added. Please close the PR after reading comments.
Good job! Keep in mind that relevant documentations should be updated too.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * Represents a particular contact element from a person.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ */
+public class Contact {
+    
+    public String value;
+    protected boolean isPrivate;
+    
+    public Contact() {
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Contact.java]: It is possible to push the constructor to the parent class (which is Contact class in this scenario)

**Review by K1ang — COMMENTED** [review]
@robinloh Some comments added. Please close the PR after reading comments.
Good job! Keep in mind that relevant documentations should be updated too.

**Pull request closed without merging**